features/c++-mode.feature

```diff
@@ -4,6 +4,14 @@ Feature: C++ specific operators
     When I turn on c++-mode
     When I turn on electric-operator-mode
 
+  Scenario: Assignment at beginning of line
+    When I type "="
+    And  I press "RET"
+    And  I go to beginning of line
+    And  I type "  ="
+    Then I should see pattern "^=$"
+    And  I should see pattern "^  =$"
```

Comment:
If you add any more tests like this: I think it would be better to have them as two separate tests (one for without indentation and one for with indentation) because it makes debugging a bit easier if tests only test one thing.

Probably not worth changing it though.

Reply:
Yeah, I agree, I'll change it